operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/ManagedInformerEventSource.java

```diff
@@ -133,4 +135,8 @@ public Stream<R> list(Predicate<R> predicate) {
     return cache.list(predicate);
   }
 
+  public void setControllerConfiguration(ControllerConfiguration<P> controllerConfiguration) {
```

Comment:
This PR is missing some tests because if you remove this method, everything still builds fine and yet, the `controllerConfiguration` is not properly initialized.

Reply:
uhh, good catch, forgot t remove that, before it was used.

Reply:
removed.

(The `InformerRelatedBehaviorITS` would fail if this would not work, although it is not very explicit test)

Reply:
Still not working right, though, because now the `controllerConfiguration` is never set… So why even record it and pass it to `initSources`?